### PR TITLE
Fixed performReadonlyTransformation

### PR DESCRIPTION
**Pull request by fonsekaean**
Issues are reported here https://github.com/silverstripe/silverstripe-tagfield/issues/99

### CODE REVIEW

**Comment by dhensby** [comment]
I've noticed that you've created this pull request from one of the "core branches" rather than a branch specifically intended for this pull request. This can cause problems if we ask you to target this PR at a different branch. At the moment you don't need to do anything but you may have to later if we need you to.

In the future please create a new branch for each individual pull request. See our [guide about contributing PRs](https://docs.silverstripe.org/en/4/contributing/code/#step-by-step-from-forking-to-sending-the-pull-request)

**Comment by dhensby** [comment]
you've got some linting issues: https://travis-ci.org/silverstripe/silverstripe-tagfield/jobs/330697822#L677

**Comment by priyashantha** [comment]
@dhensby Fixed those php linting issues. Please let's know if new PR needs to be created on a new branch rather than on master.

File: tests/TagFieldTest.php [hunk 1]
```diff
@@ -7,12 +7,15 @@
 use SilverStripe\Dev\TestOnly;
 use SilverStripe\Forms\FieldList;
 use SilverStripe\Forms\Form;
+use SilverStripe\ORM\ArrayList;
```
Comment (andrewandante) [inline tests/TagFieldTest.php]: Appears to be no longer used

File: tests/TagFieldTest.php [hunk 2]
```diff
@@ -7,12 +7,15 @@
...
 use SilverStripe\ORM\DataList;
 use SilverStripe\ORM\DataObject;
+use SilverStripe\TagField\ReadonlyTagField;
 use SilverStripe\TagField\TagField;
 use SilverStripe\TagField\Tests\Stub\TagFieldTestBlogPost;
 use SilverStripe\TagField\Tests\Stub\TagFieldTestBlogTag;
 use SilverStripe\TagField\Tests\Stub\TagFieldTestController;
+use SilverStripe\View\ArrayData;
```
Comment (andrewandante) [inline tests/TagFieldTest.php]: Appears to be no longer used

File: tests/TagFieldTest.php [hunk 3]
```diff
@@ -342,4 +345,15 @@ public function testSaveEmptySource()
             $record->Tags()->first()->ID
         );
     }
+
```
Comment (andrewandante) [inline tests/TagFieldTest.php]: Clear one of these empty lines please

**Comment by priyashantha** [comment]
@andrewandante Those updates are done.

**Pull request merged**